default.properties

```diff
@@ -42,6 +42,12 @@ headerUrl=/header/
 # Default empty string
 # georchestraStylesheet=http://my-domain-name/stylesheet.css
 
+# Config file for the header webcomponent
+# You can use this file to change the menu design, add new items, etc.
+# Full configuration cna be found here: https://github.com/georchestra/header/
+# Default empty string
+# headerConfigFile=http://my-domain-name/public/config.json
```

Comment:
shouldnt we show that urls without the fqdn should be preferred, eg `/public/config.json` ? helps for dev/staging/prod urls